cves/CVE-2017-14056.yml

```diff
@@ -90,22 +104,19 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 96f24d1bee7fe7bac08e2b7c74db1a046c9dc0de
-  note: ''
-- commit: ''
-  note: ''
+  note: Done in a batch of similar fixes for other file types with the same issue. 
 vccs:
 - commit: cb96da490265a0a798324ccce1c87dca9acbf6e9
-  note: Identified by archeogit,Identified by SZZUnleashed
-- commit: ''
-  note: ''
+  note: Identified by archeogit. Commit that added the feature
+
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes:
+upvotes: 3
```

Comment:
I give this 4 upvotes. I thought it was interesting how the absence of checking for an end of file character could lead to a denial of service attack.